alena/src/com/octonion/lesson2/Chokolate.java

```diff
@@ -0,0 +1,25 @@
+package com.octonion.lesson2;
+
+public class Chokolate extends NewYearCandy {
```

Comment:
it would be nice to have some differences between child classes of NewYearCandy

Reply:
thanks, i will think about this